Fix database bugs related to PVs for setting ADC integration windows

jamock opened the pull request:
Adjust DESC field input to be shorter because of max. length restrictions from EPICS; 
Fix typo in adc_conversion.template FLINK to FLNK

jesusvasquez333 commented on l2MpsLNApp/Db/mpsLN.substitutions:
```diff
@@ -520,13 +520,13 @@ file "waveform_stream16.template" { pattern
-{ B1_C1,         "ADC Timing Setup (Bay 1, ch 1)",       4 }
-{ B1_C2,         "ADC Timing Setup (Bay 1, ch 2)",       5 }
+{ R,                    DESC,             CH }
+{ B0_C0,         "ADC Timing B0_C0",       0 }
```
@jamock There is a copy paste in the `DESC` column here. There should not all be `B0_C0`

jesusvasquez333 commented:
@jamock Also, the `adc_conversion.template` has an undefined macro `LOCA` in the substitution file. So, I see errors like this:
```
Warning: 'db/mpsLN.db' line 7522 has undefined macros
MPLN:GUNB:MP01:1:B0_C0_COARSE_DLY_CALC.OUT Has unexpanded macro
Can't set "MPLN:GUNB:MP01:1:B0_C0_COARSE_DLY_CALC.OUT" to "TPR:$(LOCA,undefined):$(IOC_UNIT,undefined):$(INST,undefined):TRG10_SYS0_TDES" Error (512,512)
Error at or before ")" in file "db/mpsLN.db" line 7522
macLib: macro LOCA is undefined (expanding string   field(FLNK, "TPR:$(LOCA):$(IOC_UNIT):$(INST):TRG10_SYS0_TDES.PROC")
)
macLib: macro IOC_UNIT is undefined (expanding string   field(FLNK, "TPR:$(LOCA):$(IOC_UNIT):$(INST):TRG10_SYS0_TDES.PROC")
)
macLib: macro INST is undefined (expanding string   field(FLNK, "TPR:$(LOCA):$(IOC_UNIT):$(INST):TRG10_SYS0_TDES.PROC")
)
```

jamock commented:
I addressed the DESC copy paste mistake, sorry.
I updated the sioc-b084-mp02/st.cmd to include the missing macros that I used to define a forward link in the PVs I created.
